Update Daikin2's Cool mode min temp to 18C

crankyoldgit opened the pull request:
@sheppy99 has found that the daikin2 A/Cs have a min temp of 18C.
Add unit test coverage for IRDaikin2's `set/getTemp()`

Ref: #644

crankyoldgit committed:
Update Daikin2's min temp to 18C
@sheppy99 has found that the daikin2 A/Cs have a min temp of 18C.
Add unit test coverage for IRDaikin2's `set/getTemp()`

Ref: #644

sheppy99 commented:
I suspect this limitation is on the cooling modes only. I’ll verify this when I finish getting my OpenHAB cooling control rules working

crankyoldgit committed:
Change only cool mode's min temp to 18C for Daikin2
Per: https://github.com/markszabo/IRremoteESP8266/pull/658#issuecomment-478483755
@sheppy99 believes this different min temp is only in cool mode.
Adjust and add tests for this.

Ref #644

crankyoldgit commented:
@markszabo Can you Please Take Another Look? (PTAL) as I've changed it a fair bit with the last commit due to @sheppy99 's feedback.

markszabo commented:
Still LGTM :)
